.github/workflows/php.yml

```diff
@@ -40,3 +40,45 @@ jobs:
       - name: PHP CodeSniffer
         if: success() || matrix.allow_failure
         run: phpcs
+
+  test:
+    name: Unit tests with PHP ${{ matrix.php }} on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    container:
+      image: icinga/icingaweb2:latest
+      options: --user root # actions/checkout@v2 fails to install the source code without this option
+      volumes:
+        - icingaweb:/data
+
+    env:
+      PHPUNIT_VERSION: 9.5
```

Comment:
Why do we do that if we set it later to a lower version anyway?

Reply:
The later only affects running phpunit with PHP `7.2`

Reply:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategymatrixinclude